packages/cozy-sharing/src/components/SharedStatus.jsx

```diff
@@ -27,7 +28,20 @@ export const SharedStatus = ({ className, docId, recipients, link, t }) => (
         />
       </SharingTooltip>
     )}
-    {link && <LinkIcon />}
+    {link && (
+      <>
+        <span
+          style={{ fill: palette.coolGrey }}
+          data-tip
+          data-for={`linkfor${docId}`}
+        >
```

Comment:
It is not possible to put these attributes on the LinkIcon?

Reply:
BG!